packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -178,9 +178,9 @@ export const createSeatManager = (
     const zcfSeatsReallocatedOver = new Set(seats);
 
     // Ensure that all stagings are present in this reallocate call.
-    const allStagedSeatsUsed = zcfSeatToStagedAllocations
-      .keys()
-      .every(stagedSeat => zcfSeatsReallocatedOver.has(stagedSeat));
+    const allStagedSeatsUsed = [
+      ...zcfSeatToStagedAllocations.keys(),
```

Comment:
Is this change to prevent updates during iteration?  Do  you see a possible interaction that concerns you or if it's proactive, what kind of interactions are you imagining?

Reply:
No, I wasn't concerned about updates at all. Rather, `.keys()` now returns an Iterable, and Iterables have no `.every` method.